cmd/privi/main.go

```diff
@@ -49,7 +79,22 @@ func main() {
 	}
 	priviServer := privi.NewServer(adapter, privi.NewSQLiteRepo(priviDB))
 
-	mux := http.NewServeMux()
+	mux := mux.NewRouter()
```

Comment:
i think no need to import `mux`. just do `Handler: loggingMiddleware(mux)`

Reply:
oh nice